installation.rst

```diff
@@ -164,7 +164,7 @@ For the latest development version, checkout the `develop2` branch of the reposi
     $ git clone https://github.com/conan-io/conan.git conan_src
     $ cd conan_src
     $ git fetch --all
-    $ git checkout -b develop2 origin/develop2
+    $ git checkout develop2
```

Comment:
```suggestion
   # Make sure you are in the 'develop2' branch
```
The ``develop2`` is already the default branch, should be the one checked-out after ``git clone``, this is not really necessary